include/wally_transaction.h

```diff
@@ -3,6 +3,7 @@
 
 #include "wally_core.h"
 #include "wally_crypto.h"
+#include <stdbool.h>
```

Comment:
is this include needed here? can't it be added in the src that requires its use?

Reply:
Removed. That was from an earlier change.